blog-api/docker/dev/php/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# the different stages of this Dockerfile are meant to be built into separate images
```

Comment:
It is way better to use a single image for all environments and control enabling of development stuff via environment variables.

Reply:
It's better to leave them as a separated images because each project (blog, blog-api) may have different number of dependencies such as queue, databases, working with async frameworks and etc.
Having different images for each environment in the same project makes them flexible. 
I agree that it's possible to decrease the same constructions in the docker files. We can make them lighter but it's not for the first version.
You may do what you want inside this branch if you want.

Reply:
That's not about flexibility. It makes sense to have a single image:

1. Your dev matches your prod always.
2. You don't need to build image twice for staging → prod.

It's well described at https://12factor.net

Reply:
But overall, let's use what works first and adjust later.